### PR TITLE
feat: add API cohort 

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
This PR adds a mechanism for getting cohort information using Xblock Architecture

Taken from README.rst

Cohort API used to get information about the student's cohort.

Intructions on how to use:

1. Include this component in your course. You can include it in any course unit.
2. Save the Block ID.
3. Using it, you can reconstruct the handler URL used to get cohort information. You can follow this pattern:
   currentDomain/courses/courseID/xblock/xblockID/handler/get_user_cohort
4. Make a POST request to the handler as follows (using javascript):

```
    var headers = {
    "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8",
    "x-csrftoken": getCSRFToken(),

    }

    return fetch(
    `domain/courses/${courseIdSafe}/xblock/${xblockIDSafe}/handler/get_user_cohort`,
    {
        method: "POST",
        credentials: 'include',
        body: JSON.stringify({}),
        headers: headers
    },
    )
    .then(function (response) {
        if (!response.ok) throw Error(response.statusText);
        return response.json();
    })
        .then(function (response) {
        return response;
    })
        .catch(function (error) {
        console.warn(error);
    });
    }
```

Then you'll get the cohort information. For now, we are only sending the name of the cohort.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,5 +1,44 @@
...
+
+Intructions on how to use:
+
+1. Include this component in your course. You can include it in any course unit.
+2. Save the Block ID.
+3. Using it, you can reconstruct the handler URL used to get cohort information. You can follow this pattern:
+   currentDomain/courses/courseID/xblock/xblockID/handler/get_user_cohort
+4. Make a POST request to the handler as follows (using javascript):
+
+.. code-block:: javascript
```
Comment (MoisesGSalas) [inline README.rst:13]:
```suggestion
.. code-block:: javascript

```
One blank line is needed for the block to be rendered.

File: README.rst [hunk 2]
```diff
@@ -1,5 +1,44 @@
...
+        if (!response.ok) throw Error(response.statusText);
+        return response.json();
+    })
+        .then(function (response) {
+        return response;
+    })
+        .catch(function (error) {
+        console.warn(error);
+    });
+    }
```
Comment (MoisesGSalas) [inline README.rst:41]: Extra brace.

**Review by MoisesGSalas — APPROVED** [review]

File: cohort_api_xblock/static/html/cohort_api_xblock.html [hunk 3]
```diff
@@ -1,5 +0,0 @@
-<div class="cohort_api_xblock_block">
```
Comment (andrey-canon) [inline cohort_api_xblock/static/html/cohort_api_xblock.html:1]: will this unit be visible for the students ? so there will be  an empty component in the middle of the course content ...

Comment (mariajgrimaldi) [reply cohort_api_xblock/static/html/cohort_api_xblock.html:1]: Yes, unfortunately, it's an empty component. But if you add it with another component, it's seamless

**Review by felipemontoya — APPROVED** [review]
In terms of architecture I'm good with this

**Pull request merged**